RocketChat/Rocket.Chat
# fix: Custom OAuth settings are not visible

## Proposed changes (including videos or screenshots)


 - Ensure that custom OAuth settings have their `public` field defined so that they're displayed in the UI.



## Issue(s)


## Steps to test or reproduce

Create a custom OAuth in an older RC version (I'd suggest 4.x or prior) and check if its settings have the `public` field in the DB (or erase it manually). Upgrade to 6.x and try to check the custom OAuth settings in the UI.
**Current behavior:** the settings are not displayed;
**New behavior:** the settings have been added the `public` field in the DB (with the `false` value) and, hence, are visible in the UI.

## Further comments

Introduced in #27653
SUP-208
